api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -970,8 +970,9 @@ message DatabaseCSRRequest {
   bytes CSR = 1 [(gogoproto.jsontag) = "csr"];
   // ClusterName is the name of the cluster the request is for.
   string ClusterName = 2 [(gogoproto.jsontag) = "cluster_name"];
-  // SignWithDatabaseCA deprecated.
+
   reserved 3;
+  reserved "SignWithDatabaseCA";
```

Comment:
Reserving the names is a good call, I often forget that. Is this something we can lint for?

Reply:
It's gonna be linted as part of `buf breaking` in `WIRE_JSON` mode, you're only allowed to delete fields if you reserve both the number and the name.